ui/src/components/Filter/Filter.tsx

```diff
@@ -5,7 +5,7 @@ import TimesIcon from '@patternfly/react-icons/dist/js/icons/times-icon';
 import './Filter.css';
 
 interface Filterable {
-  id: number;
+  id?: number;
```

Comment:
why is this optional?

Reply:
We don't have `id` in `kind` and then test fails hence added `id` as optional